src/vivarium/framework/results/context.py

```diff
@@ -119,10 +119,11 @@ def gather_results(
             else:
                 filtered_pop = population
             if filtered_pop.empty:
-                yield None
+                yield None, None
             else:
-                if not list(stratifications):  # Handle situation of no stratifications
-                    pop_groups = filtered_pop.groupby(lambda _: True)
```

Comment:
Rajan: let's go back to making a pop_group of everything - even no stratificatons. Call it "all" or whatever.

Reply:
✅